blueocean-dashboard/src/main/js/components/karaoke/services/LiveUpdate.js

```diff
@@ -0,0 +1,184 @@
+console.log('lu');
+
+import React from 'react';
+import { action, observable } from 'mobx';
+import { logging, sseConnection } from '@jenkins-cd/blueocean-core-js';
+import { KaraokeConfig, KaraokeSpeed } from '../';
+import debounce from 'lodash.debounce';
+
+const logger = logging.logger('io.jenkins.blueocean.dashboard.following.liveUpdate');
+
+export class LiveUpdate {
```

Comment:
trying to separate the 'followAlong' logic from the components, so they just render; this wasn't all the way complete